README.md

```diff
@@ -72,10 +95,9 @@ hugo server
 
 3. Navigate to `http://localhost:1313/`
 
-
 ## Update docs
 
-1. Fork repo into your account
+1. Ensure you are in your forked repo
 1. Create new branch
```

Comment:
While you are updating this, can you update the bulleted numbers. It looks as though each step is `#1`

Reply:
Done